src/main/resources/application.properties

```diff
@@ -3,3 +3,4 @@ spring.datasource.username=
 spring.datasource.password=
 spring.jpa.hibernate.ddl-auto=update
 spring.profiles.active=DEV
+weatherservice.addlocation.permissionrole=ADMIN
```

Comment:
do you want to change a name value of this role? if not, then just create a class with a static constant value